test/k8sT/manifests/microscope.yaml

```diff
@@ -69,6 +69,6 @@ spec:
   - args:
     - sleep
     - "100000"
-    image: docker.io/cilium/microscope:1.0.1
+    image: docker.io/cilium/microscope:ci
```

Comment:
Can't we have a version? something like `ci-x.y.z`? or `x.y.z-ci`

Reply:
we can, will do

Reply:
done